vision/segmentation/data/intel.json

```diff
@@ -64,7 +64,7 @@
                     }
                 },
                 "intrinsic_extrinsic": {
-                    "color_intrinsics": {
+                    "color_intrinsic_parameters": {
```

Comment:
Was this file used anywhere other `align_test.go`? If not, could we delete? I'm ok with leaving it here as well if you think it might be useful in the future.

Reply:
good point, we can delete

Reply:
deleted